unittest/unit/build.js

```diff
@@ -509,6 +509,45 @@ describe('build', () => {
       assert.equal(interfaces[1].name, 'XSLTProcessor');
     });
 
+    it('WindowOrWorkerGlobalScope remains separate', () => {
+      const specIDLs = {
+        first: WebIDL2.parse(
+          `[Exposed=Window]
+             interface Window {
+               readonly attribute boolean imadumdum;
+             };`
+        ),
+        second: WebIDL2.parse(
+          `[Exposed=Window]
+             interface mixin WindowOrWorkerGlobalScope {
+               undefined atob();
+             };
+
+             Window includes WindowOrWorkerGlobalScope;`
+        )
+      };
+      const {ast, globals} = flattenIDL(specIDLs, customIDLs);
+      assert.lengthOf(ast, 3);
+      assert.lengthOf(globals, 1);
+
+      assert.equal(ast[0].name, 'Window');
+      assert.lengthOf(ast[0].members, 1);
+      assert.containSubset(ast[0].members[0], {
+        type: 'attribute',
+        name: 'imadumdum'
+      });
```

Comment:
This should already be tested by some other test. I don't think testing the stuff that remains in `ast` in depth here is necessary.